app/steps/ui/tasklist/includes/documents.html

```diff
@@ -2,14 +2,21 @@
 
 <li class="task-item">
     <span class="heading-medium">{{content.documentTask}}</span>
+
     {% if fields.DocumentsTask.value.status == 'complete' and previousTaskStatus !== 'started' %}
         <span class="progress-status-tag">{{content.taskCompleteTag | upper}}</span>
+    {% else %}
+        <span class="task-unavailable">{{content.taskUnavailable | safe}}</span>
     {% endif %}
```

Comment:
I was going to refactor these pages when I got the content through, and it will be clearer if the paragraphs need to be included or not and should make refactoring easier. Is this OK?

Reply:
This is done, I've put the logic in a separate macro.